src/graphx/graphx.h

```diff
@@ -1063,19 +1081,21 @@ typedef enum {
 
 /**
  * Sets the text foreground color.
+ * Note that if you haven't called
```

Comment:
seems to be missing the end of this statement

Reply:
Yeah, sorry. I moved it to the @note block at the end